### PR TITLE
Make squoosh image customization work

### PR DESCRIPTION
**Pull request by yiqunw700**
ImageWithContext customization still doesn't work.

Towards:  #1719

### CODE REVIEW

File: designcompose/src/main/java/com/android/designcompose/FrameRender.kt [hunk 1]
```diff
@@ -852,12 +852,10 @@ internal fun squooshShapeRender(
         for (fill in shapePaths.fills) {
             drawContext.canvas.clipPath(fill)
         }
-        /* XXX XXX RALPH
```
Comment (iamralpht) [inline designcompose/src/main/java/com/android/designcompose/FrameRender.kt:855]: Hm, I wonder who did this. 🤔

File: integration-tests/validation/src/main/java/com/android/designcompose/testapp/validation/examples/AllExamples.kt [hunk 2]
```diff
@@ -74,6 +74,7 @@ val EXAMPLES: ArrayList<Triple<String, @Composable () -> Unit, String?>> =
         Triple("Text Elide", { TextElideTest() }, TextElideTestDoc.javaClass.name),
         Triple("Text Inval", { TextResizingTest() }, TextResizingTestDoc.javaClass.name),
         Triple("Styled Text Runs", { StyledTextRunsTest() }, StyledTextRunsDoc.javaClass.name),
+        Triple("Shared Customization", { ModuleExample() }, ModuleExampleDoc.javaClass.name),
```
Comment (iamralpht) [inline integration-tests/validation/src/main/java/com/android/designcompose/testapp/validation/examples/AllExamples.kt:77]: 🥳

File: designcompose/src/main/java/com/android/designcompose/FrameRender.kt (hunk 1 above)
Comment (yiqunw700) [reply designcompose/src/main/java/com/android/designcompose/FrameRender.kt:855]: 🤔

File: integration-tests/validation/src/main/java/com/android/designcompose/testapp/validation/examples/AllExamples.kt (hunk 2 above)
Comment (yiqunw700) [reply integration-tests/validation/src/main/java/com/android/designcompose/testapp/validation/examples/AllExamples.kt:77]: 🥳

**Pull request merged**